pkg/yang/yang.go

```diff
@@ -646,7 +650,7 @@ type Refine struct {
 	Parent     Node         `yang:"Parent,nomerge"`
 	Extensions []*Statement `yang:"Ext"`
 
-	Default     *Value   `yang:"default"`
+	Defaults    []*Value `yang:"default"`
```

Comment:
This is a backwards incompatible change, that needs to be handled by downstream consumers too. 

We have had some discussion before on this: https://github.com/openconfig/goyang/issues/51

The conclusion was that we would have a `Default` and `Defaults` field, and this would allow handling of these cases in a backwards compatible way. Can we do this here so that we don't break the existing users?